### PR TITLE
Update evergreen build configs for 25.lts

### PR DESCRIPTION
**Pull request by oxve**
Add sbversion 16, remove sbversion 13.

b/293645066

### CODE REVIEW

File: .github/config/evergreen-arm-softfp.json [hunk 1]
```diff
@@ -13,8 +13,9 @@
   },
   "platforms": [
     "evergreen-arm-softfp",
+		"evergreen-arm-softfp-sbversion-14",
```
Comment (kaidokert) [inline .github/config/evergreen-arm-softfp.json:16]: Something funny with formatting here ( tabs vs spaces ? )

File: .github/config/evergreen-x64.json [hunk 2]
```diff
@@ -5,9 +5,9 @@
   "on_host_test_shards": ["0", "1", "2", "3", "blackbox", "wpt"],
   "platforms": [
     "evergreen-x64",
-    "evergreen-x64-sbversion-15",
 		"evergreen-x64-sbversion-14",
-    "evergreen-x64-sbversion-13"
+		"evergreen-x64-sbversion-15",
```
Comment (kaidokert) [inline .github/config/evergreen-x64.json:9]: Formatting thing here too

**Review by kaidokert — APPROVED** [review]

File: .github/config/evergreen-arm-softfp.json (hunk 1 above)
Comment (oxve) [reply .github/config/evergreen-arm-softfp.json:16]: Oh yeah, I fixed a couple of other instances. I must've missed this one.

Whoever added sbversion-14 did it all with tabs.

File: .github/config/evergreen-x64.json (hunk 2 above)
Comment (oxve) [reply .github/config/evergreen-x64.json:9]: Done

File: .github/config/linux.json [hunk 3]
```diff
@@ -7,9 +7,9 @@
     "linux-x64x11",
     "linux-x64x11-egl",
     "linux-x64x11-skia",
-    "linux-x64x11-sbversion-13",
     "linux-x64x11-sbversion-14",
-    "linux-x64x11-sbversion-15"
+    "linux-x64x11-sbversion-15",
```
Comment (kaidokert) [inline .github/config/linux.json:11]: doesn't Raspi-2 also need updates here? We build it at least for one sbversion

**Review by y4vor — APPROVED** [review]

**Review by kaidokert — APPROVED** [review]

File: .github/config/linux.json (hunk 3 above)
Comment (oxve) [reply .github/config/linux.json:11]: Done

**Pull request merged**